packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -94,29 +90,9 @@ export default function NavigationPlaceholder( {
 										popoverProps={ { isAlternate: true } }
 									>
 										{ () => (
-											<ExistingMenusOptions
```

Comment:
Can the `ExistingMenusOptions` component be entirely removed now?

Reply:
Ah, it is removed. Ignore me :D

Reply:
It is now you asked 😄